web-api/openapi/back_office.yaml

```diff
@@ -303,3 +326,19 @@ components:
           maxItems: 1000
           items:
             $ref: './schemas/document.yml#/DocumentId'
+    DocumentCandidate:
+      type: object
+      required: [id]
+      properties:
+        id:
+          $ref: './schemas/document.yml#/DocumentId'
+    DocumentCandidatesRequest:
+      type: object
+      required: [documents]
+      properties:
+        documents:
+          type: array
+          minItems: 0
+          maxItems: 1e+6
```

Comment:
exactly, the linter forced me to add a `maxItems`. i set it to `1e6` mostly arbitrarily, since json doesn't allow for infinity. is there a way to ignore the linter for this item? or is there a way to say "unlimited but bound by general size limits"?